anime_tracker/src/models/animeSearch.ts

```diff
@@ -0,0 +1,149 @@
+export type OverviewArray = Overview[]
```

Comment:
naming (again):
is this type (OverviewArray) necessary? Overview[] is even shorter. and if you want this type Overviews (plural) is a better name, perhaps

Reply:
adjusted